### ISSUE TITLE
[Feature Request] Remember downloaded/seen ebooks

### ISSUE THREAD

**Issue by Cwpute** [Pool-Of-Tears/Myne#80]
### Checklist

- [X] This issue contains only one feature request. I will open one issue for every feature I want to request.
- [X] I checked, but didn't find any duplicates (open OR closed) of this issue in the repo.
- [X] This feature is merely an UI/UX update.
- [X] This feature is not going to conflict with many of the existing features/options.

### Is your feature request related to a problem? Please describe.

I was searching for tales myths and legends and dovnloaded a good bunch of books. Unfortunately i couldn't remember which ones i did download…

### Describe the solution you'd like

The app could remember which ebook pages have been seen, or even downloaded.
This could be seen both on the ebook list and ebook page with an eye icon 👁️ for a page seen, and a dowload icon 📥✅ for downloaded ones.

Further down the line, it could even become a sorting filter to hide seen/downloaded ebooks.

### How will you/everyone benefit from this feature?

This will help anyone willing to read new books to sort out the ones they have already seen before.

**Comment by starry-shivam** [comment]
Good idea, I'll try to implement something like this if i get enough time.
